lib/vaahextendflutter/widgets/atoms/input_text.dart

```diff
@@ -48,19 +49,20 @@ class InputText extends StatelessWidget {
     this.suffixOnTap,
     this.minLines,
     this.maxLines,
+    this.inputBorder = InputBorderType.outline,
   });
 
   @override
   Widget build(BuildContext context) {
     return TextFormField(
       decoration: InputDecoration(
         contentPadding: padding,
-        border: border(AppTheme.colors['black']!.shade400),
-        enabledBorder: border(AppTheme.colors['black']!.shade400),
-        disabledBorder: border(AppTheme.colors['black']!.shade300),
-        focusedBorder: border(AppTheme.colors['black']!.shade400),
-        errorBorder: border(AppTheme.colors['danger']!.shade400),
-        focusedErrorBorder: border(AppTheme.colors['danger']!.shade400),
+        border: getInputBorder(inputBorder, AppTheme.colors['black']!.shade400),
+        enabledBorder: getInputBorder(inputBorder, AppTheme.colors['black']!.shade400),
+        disabledBorder: getInputBorder(inputBorder, AppTheme.colors['black']!.shade400),
+        focusedBorder: getInputBorder(inputBorder, AppTheme.colors['black']!.shade400),
+        errorBorder: getInputBorder(inputBorder, AppTheme.colors['black']!.shade400),
```

Comment:
i think we shouldn't hard code border color, we should get dev's input as argument, what are your thoughts?

Reply:
Yes, I agree. That will give dev's flexibility to choose border color if there's a requirement in their design.

Reply:
Yep, now that's correct. cheers!